models/userDao.js

```diff
@@ -49,9 +49,125 @@ const checkMail = async (email) => {
   );
   return !!parseInt(result.registerd);
 };
+//토큰관련
+const getUserId = async (email) => {
```

Comment:
```suggestion
const getUserId = async (email) => {
```
실제 SELECT 해오는 data는 email인데 getUserId라고 작성되어 있네요! 
다만 위에서 이전 코멘트에 드린 리뷰처럼 email이 아닌 id를 활용해야하기때문에 메소드 네임이 수정되기 보다는 query문이 수정될 수 있도록 해주세요.

Reply:
네 수정했습니다 !